libs/markets/src/lib/markets-provider.ts

```diff
@@ -199,7 +199,7 @@ export const allMarketsWithLiveDataProvider = makeDerivedDataProvider<
     return data.find(
       (market) =>
         market.id ===
-        (parts[1].delta as MarketDataUpdateFieldsFragment).marketId
+        (parts[1].delta as MarketDataUpdateFieldsFragment)?.marketId
```

Comment:
not great but sorted out this issue 
<img width="1267" alt="Screenshot 2023-09-04 at 18 39 57" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/46ce57fb-f490-40df-9459-291417b355dc">